ToDoList/DTOs/Todo/UpdateTodoDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TodoList.DTOs.Todo;
+
+public class UpdateTodoDTO : CreateTodoDTO
+{
+    public int Id { get; set; }
+    public bool IsDone { get; set; }
```

Comment:
we should also update Title and Description of a Todo

Reply:
The Title and Description is already part of the `CreateTodoDTO `which the `UpdateTodoDTO `will inherit